packages/venia-ui/lib/queries/getNavigationMenu.graphql

```diff
@@ -1,4 +1,8 @@
 query navigationMenu($id: Int!) {
+    storeConfig {
+        id
+        category_url_suffix
```

Comment:
We should use the `url_suffix` field off the category query response and not the store config, for now.